test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -632,6 +632,7 @@ def __convert_flag_to_numeric(flag):
         ATR_DFLAG_OPWR = 0x08
         ATR_DFLAG_MGRD = 0x10
         ATR_DFLAG_MGWR = 0x20
+        ATR_DFLAG_MOM = 0x400
```

Comment:
Just wondering why 0x400 and not 0x40,  it makes more sense to utilize bit in incremental order.

Reply:
@dilip-krishnan existing definitions match with what we have in pbs_internal.h. For example we have ATR_DFLAG_RASSN = 0x4000. I tried to follow it and defined the same value for ATR_DFLAG_MOM as defined in pbs_internal.h.

Reply:
@ashwathraop please add a comment to this effect. Also note that the value of ATR_DFLAG_RASSN is defined as 0x4000 and not 0x400 in pbs_internal.h.

Reply:
pbs_internal.h has value of ATR_DFLAG_RASSN is 0x4000 and ATR_DFLAG_MOM is 0x400. That is how I defined. I am not sure if I follow your comment.

Here is the list of flags we used in the snaputils file:
        ATR_DFLAG_USRD = 0x01
        ATR_DFLAG_USWR = 0x02
        ATR_DFLAG_OPRD = 0x04
        ATR_DFLAG_OPWR = 0x08
        ATR_DFLAG_MGRD = 0x10
        ATR_DFLAG_MGWR = 0x20
        ATR_DFLAG_MOM = 0x400
        ATR_DFLAG_RASSN = 0x4000
        ATR_DFLAG_ANASSN = 0x8000
        ATR_DFLAG_FNASSN = 0x10000
        ATR_DFLAG_CVTSLT = 0x20000

Reply:
Sorry @ashwathraop, I was looking at the wrong line. Your definition of ATR_DFLAG_MOM is correct. Please do add a comment indicating where the value originated.

Reply:
@mike0042 I have added the comment to refer pbs_internal.h